Ola_alex.html

```diff
@@ -0,0 +1,78 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>
+      My First Website
+    </title>
+  </head>
+  <body itemscope itemtype="https://schema.org/Person"> //This can't be used for two people but we didn't tell you to only write for one person beforehand so this is our bad
+    <header>
+        <a href="#maincontent"> Skip to content</a>
+      <nav>
+          <a href="#">Home</a> |
+          <a href="#">Contact</a>
+      </nav>
+      
+    <h1><div itemProp="name">Name: Olamide & Alex</div> 
+    <div itemProp ="jobTitle"> Job title: Superheroes</div>
+    <div itemProp ="worksFor"> Company: AND Digital</div>
+      
+    </h1>
+    </header>
+  <main id="maincontent">
+    <figure>
+      <img src="images/cropped AND photo.png" alt="Professional picture" height="120" width="120">
+      <figcaption>A really good picture</figcaption>
+    </figure>
+    
+    <section>
+        <h2>About</h2>
+        <h3>I am a superdev with astonishing java superpowers. :heart: Dekker</h3> //This isn't a heading - probably a paragraph
```

Comment:
This isn’t a heading, this is a paragraph of text  to support the ‘about’ section.